examples/all-clusters-app/esp32/README.md

```diff
@@ -160,73 +160,71 @@ your network configuration. To erase it, simply run.
 
           $ idf.py -p /dev/tty.SLAB_USBtoUART erase_flash
 
-### Setting up Python Controller
+-   Once ESP32 is up and running, we need to set up a device controller to
+    perform
+-   commissioning and cluster control.
 
-Once ESP32 is up and running, we need to set up a device controller to perform
-commissioning and cluster control.
+### Setting up chip-tool
 
--   Set up python controller.
+See [the build guide](../../../docs/guides/BUILDING.md#prerequisites) for
+general background on build prerequisites.
 
-           $ cd {path-to-connectedhomeip}
-           $ ./scripts/build_python.sh -m platform
+Building the example:
 
--   Execute the controller.
+```
+$ cd examples/chip-tool
+
+$ rm -rf out
 
-           $ source ./out/python_env/bin/activate
-           $ chip-device-ctrl
+$ git submodule update --init
 
-### Commissioning over BLE
+$ source third_party/connectedhomeip/scripts/activate.sh
```

Comment:
This action is already listed in setting up the `Setting up the environment` section, also in the below file.

Reply:
done